### DISCUSSION TITLE
Release Partners 9/16/2021

### DISCUSSION THREAD

**Discussion by joshblack** [carbon-design-system/carbon#9675] [Release Feedback]
This week, we're planning on releasing Beta 2. In this session, we'll talk about what's making its way in this release and what you can expect from our team next.

**Agenda**

- Scope of the beta 2 release
- Scope and timeline of our next release
- v11 Communication
- Research
- Q&A

## Beta 2 Release

- We will officially release our beta early next week, we're looking to get in the final remaining PRs that we weren't able to get in this sprint
- Release scope
  - Certain prop changes are making their way into v10 (some are in v10.44, some are targeting v10.45), including
    - Changes to `className`
    - Changes to event handlers (`handle*` to `on*`)
    - Changes to size prop
  - This release will **not** include all of the prop changes we are looking to make
  - Early versions of components, including
    - Popover
    - Tooltip
    - Notification
    - ContentSwitcher
  - Access to our icon changes under `@carbon/react/icons`
  - Bug fixes from our last release cycle, namely with theming and our compatibility theme
    - Included filing an issue with Sass as we encountered our first bug with Dart Sass
  - The `bx` prefix has been changed to `cds`

## Scope of next release

- The target date for the next release is **October 14th, 2021**
- We are targeting this release as our first Release Candidate for v11
- We are trying to target the following for this release:
  - All expected changes to props in our React components
  - All expected updates to existing components, notably:
    - Tooltip, Popover
    - Tabs
  - The initial set of supported migrations from `@carbon/upgrade`, including migrations for:
    - Switching packages and versions
    - Common prop changes
    - [Stretch] Sass import changes
  - Migration guidance
    - Expanded v11 migration page
    - Sample App

## v11 Communication

- We're trying to publish a Medium article around every other week on v11-related migration topics
- Our next article in the pipeline will talk about updating the tutorial to use the latest beta
- We're targeting future content on theming, grid, react changes, and more
- This content should overlap with workshops / other content we are aiming to produce for migration
- Are there any topics that you would like to see us cover or write about? 

## Research

- Similar to last time, we want to connect with teams to see what their experience is like updating to the newest beta
- This experience helps us focus on what to prioritize in terms of documentation and tooling

## Q&A

Any questions about the upcoming beta 2 release and the future Release Candidate release?

**Comment by adrianamorales** [comment]
@joshblack Are there any UX or visual changes to popover, tooltip, notification,  or content switcher? I'm wondering if I need to make designers in Cloud aware of it.

**Reply by joshblack** [reply]
Hey @adrianamorales! So sorry for the delay. As far as I'm aware, the changes will be mostly around behavior but there could be some visual changes with notification. Let me cc @kingtraceyj and @tay1orjones who have been working on it 👀
